test/test_sqlalchemy_bigquery.py

```diff
@@ -199,9 +199,7 @@ def test_engine_with_dataset(engine_using_test_dataset):
     rows = table_one_row.select().execute().fetchall()
     assert list(rows[0]) == ONE_ROW_CONTENTS_EXPANDED
 
-    # Table name shouldn't include dataset
-    with pytest.raises(Exception):
-        table_one_row = Table('test_pybigquery.sample_one_row', MetaData(bind=engine_using_test_dataset), autoload=True)
```

Comment:
This is another change in behavior, however seems to me that this is just another bug fix.

Reply:
That's correct.  The lookup code is the same between `Table()` and `has_table()`, so the behavior changed in both places.   I agree that it is also a bug fix.